website/next.config.js

```diff
@@ -4,15 +4,33 @@ const { i18n } = require('./next-i18next.config.js')
 
 const { withGuildDocs } = require('@guild-docs/server')
 
-const { getRoutes } = require('./routes.ts')
+const { getRoutes, getTutorialRoutes } = require('./routes.ts')
 
 const withBundleAnalyzer = require('@next/bundle-analyzer')({
   enabled: process.env.ANALYZE === 'true',
 })
 
 module.exports = withBundleAnalyzer(
   withGuildDocs({
+    env: {
+      // This is a pre-serialized version of the tutorial routes that prevents calculating the routes on production
+      SERIALIZED_TUTORIAL_MDX_ROUTES: JSON.stringify(getTutorialRoutes()),
+    },
     i18n,
     getRoutes,
+    redirects: () => {
+      return [
+        {
+          source: '/docs',
+          destination: '/docs/quick-start',
+          permanent: true,
+        },
+        {
+          source: '/tutorial',
+          destination: '/tutorial/00-introduction',
```

Comment:
Can/should we add numbering to the chapters on the sidebar?
Can make it nicer for people to keep track on where they are and feel like they are progressing

Reply:
Great idea, but somehow guild-docs strips out numbers 😕 I created https://github.com/the-guild-org/the-guild-docs/issues/507 for tracking this.